cpp/merge_print_and_online.cc

```diff
@@ -493,7 +493,8 @@ void UpdateMergedPPNs(MARC::Record * const record, const std::set<std::string> &
     if (not max_publication_year.empty())
         zwi_subfields.addSubfield('y', max_publication_year);
 
-    record->replaceField("ZWI", zwi_subfields);
+    record->deleteFields("ZWI");
+    record->appendField("ZWI", zwi_subfields);
```

Comment:
Do we lose information this way? (e.g. can it happen that the existing ZWI contains information about formerly merged PPNs that are not included in the given merged_ppns array?)